sunpy/lightcurve/sources/stereo.py

```diff
@@ -0,0 +1,906 @@
+			data.add_column(Column(data = data_modify, name='Datetime'),0)
+		
+		#For all other types of data since they share the same format    
+		else:
+			#Storing data columns in recognizable variables
+			hour_col = data['col3']
+			minutes_col = data['col4']
+			seconds_col = data['col5']
+
+			for i in range(len(data)):
+				data_modify.append(datetime(year_col[i],1,1,hour_col[i],minutes_col[i],seconds_col[i]) + timedelta(days = int(day_of_year_col[i]-1)))
+
+			header = ['Datetime'] + header[5:]
+
+			data.remove_columns(['col{}'.format(i) for i in range(1,6)])
+			data.add_column(Column(data = data_modify, name='Datetime'),0)
+		
+		# Converting from astropy.table.Table to pandas.Dataframe
+		# to_pandas() bound method is only available in the latest development build and none of the stable
+		data = _to_pandas(data)
+		
+		n = len(header)
+		return [OrderedDict(zip(range(n), header)), data]
+
+
+class SITLightCurve(LightCurve):
+	"""
+	SIT LightCurve. Provides SIT data back to 2007-07.
+	Most recent data is usually available one or two days late. 
+	Currently the LightCurve supports only Single File Load !!
+
+	Parameters
+	----------
+	timerange: sunpy.time.TimeRange
+		time range for which data is to be downloaded.
+		Example value -  TimeRange('2007-01-01','2015-03-01')   
+
+	stereo_spacecraft: string   
+		Default value - ahead
+		Possible values - ahead, behind    # corresponding to spacecraft location
+
+	specie:  string
+		Default value - 4He
+		Possible values - 4He, Fe, H, O
+
+	duration_of_average: string
+	Default value - 15min
+		Possible values - 1min, 10min, 1hr, 1day        #corresponding to duration over which data is averaged
+
+
+	Examples
+	--------
+	>>> import os
+	>>> import sunpy.data.test
+	>>> filepath = sunpy.data.test.rootdir
+	>>> from sunpy import lightcurve as lc
+	>>> sit = lc.SITLightCurve._parse_txt(os.path.join(filepath , 'sit/SIT_Ahead_10min_H_2007_01.txt'))
+	>>> sit = lc.SITLightCurve(sit[1],sit[0])
+	>>> sit.peek()
+
+
+	References
+	----------
+	| http://www.srl.caltech.edu/STEREO/Public/SIT_public.html
+
+	"""
+
+	def peek(self, title="SIT elemental Intensities"):
+		"""Plots SIT light curve in the usual manner"""
+
+		figure = plt.figure()
+		axes = plt.gca()
+
+		dates = matplotlib.dates.date2num(self.data['DateTime'].astype(datetime))
+
+		num_energy_bins = (len(self.header.values())-2)/2
+		colors = ['Green','Red','Chocolate', 'Blue','SeaGreen','Tomato','SlateBlue','Orange','Purple','Magenta','MediumVioletRed']
+
+		figure.delaxes(ax)
+		axes = figure.add_axes([0.1, 0.15, 0.55, 0.8])
+
+		for i,line in enumerate(self.header.values()):
+			if i >= 2 and i <= num_energy_bins + 1:
+				axes.plot_date(dates, self.data[line].ffill(), '-',
+					 label=line[1:20], color=colors[i-2], lw=0.5)
+		
+		axes.set_yscale("log",nonposy='mask')
+		axes.set_ylim(1e-3, 1e+3)
+		axes.set_title(title + ' : ' + self.header.values()[-1][:self.header.values()[-1].index(' ')])
```

Comment:
Could you use the `' {title} : {species}'.format(title=title, species=...` sintax, please.